frontend/src/components/custom-tools/prompt-card/PromptCardItems.jsx

```diff
@@ -1,28 +1,72 @@
 import PropTypes from "prop-types";
-import { LeftOutlined, RightOutlined, SearchOutlined } from "@ant-design/icons";
+import {
+  ArrowsAltOutlined,
+  CheckCircleOutlined,
+  DatabaseOutlined,
+  ExclamationCircleFilled,
+  InfoCircleOutlined,
+  PlayCircleFilled,
+  PlayCircleOutlined,
+  SearchOutlined,
+} from "@ant-design/icons";
 import {
   Button,
   Card,
+  Col,
   Collapse,
   Divider,
+  Image,
+  Radio,
+  Row,
   Select,
   Space,
   Spin,
-  Tag,
+  Tooltip,
   Typography,
 } from "antd";
 import { useEffect, useRef, useState } from "react";
 
-import { displayPromptResult } from "../../../helpers/GetStaticData";
+import {
+  displayPromptResult,
+  getFormattedTotalCost,
+} from "../../../helpers/GetStaticData";
 import { SpinnerLoader } from "../../widgets/spinner-loader/SpinnerLoader";
 import { EditableText } from "../editable-text/EditableText";
 import { TokenUsage } from "../token-usage/TokenUsage";
 import { useCustomToolStore } from "../../../store/custom-tool-store";
 import { Header } from "./Header";
+import CheckableTag from "antd/es/tag/CheckableTag";
+import { useAxiosPrivate } from "../../../hooks/useAxiosPrivate";
+import { useSessionStore } from "../../../store/session-store";
+import { motion, AnimatePresence } from "framer-motion";
+import { OutputForIndex } from "./OutputForIndex";
+import { useExceptionHandler } from "../../../hooks/useExceptionHandler";
+import { useAlertStore } from "../../../store/alert-store";
 
 const EvalBtn = null;
 const EvalMetrics = null;
 
+function useWindowDimensions() {
+  const [windowDimensions, setWindowDimensions] = useState({
+    width: window.innerWidth,
+    height: window.innerHeight,
+  });
+
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions({
+        width: window.innerWidth,
+        height: window.innerHeight,
+      });
+    }
+
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  return windowDimensions;
+}
```

Comment:
@jagadeeswaran-zipstack Can you move this inside the hooks folder?

Reply:
Done :+1: